### DISCUSSION TITLE
how can carry out other event outside the <Drawer> component when <Drawer> is open?

### DISCUSSION THREAD

**Discussion by zhoudalang** [chakra-ui/chakra-ui#7077] [Q&A]
how can i carry out other event outside the <Drawer> component when <Drawer> is open? please.  I can't click any other element  when I open Drawer and set **closeOnOverlayClick** false.

https://codesandbox.io/s/strange-mahavira-0vspiy?file=/src/App.js

**Comment by Tisks** [comment]
I don't see any code with a Drawer component in that codesandbox

**Reply by zhoudalang** [reply]
sorry, i just post a new link.
